lib/breakers/statsd_plugin.rb

```diff
@@ -2,12 +2,15 @@
 
 module Breakers
   class StatsdPlugin
-    def get_tags(request)
+    def get_tags(request, response = nil)
       tags = []
       if request
```

Comment:
can request ever really be nil or false seeing as it is always passed in? 🤔 

I'm not sure if this condition is necessary.

Reply:
Good point, it may just there to prevent `request.method` from throwing if for whatever reason `request` is *nil*.  I don't think it's a bad gate though.